src/libs/ajax/leonardo/Apps.ts

```diff
@@ -77,13 +77,16 @@ export const Apps = (signal) => ({
       },
     };
   },
-  listAppsV2: async (workspaceId: string): Promise<ListAppResponse[]> => {
-    const res = await fetchLeo(`api/apps/v2/${workspaceId}`, _.mergeAll([authOpts(), appIdentifier, { signal }]));
+  listAppsV2: async (workspaceId: string, labels: LeoResourceLabels = {}): Promise<ListAppResponse[]> => {
```

Comment:
Added ability to list-by-labels, which lets the UI pass the `role=creator` param to filter apps on the server side (we were already doing this for v1 apps but not v2).